pkg/util/runtime/features.go

```diff
@@ -140,7 +140,7 @@ var (
 		FeaturePortPolicyNone:                   false,
 
 		// Dev features
-		FeatureAutopilotPassthroughPort: false,
```

Comment:
Did you mean to include this? This would be the equivalent of promoting it to Beta.

I think after this change goes in you can move it up to Alpha, though (follow the instructions above to do that).